src/components/ui/Checkbox/checkbox.scss

```diff
@@ -1,104 +1,153 @@
 @import "src/styles/scaffold";
 
-[data-store-checkbox] {
+[data-fs-checkbox] {
+  // --------------------------------------------------------
+  // Design Tokens for Checkbox
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-checkbox-width                        : 1.25rem;
+  --fs-checkbox-height                       : 1.25rem;
+
+  --fs-checkbox-border                       : var(--fs-border-width) solid var(--fs-border-color);
+  --fs-checkbox-border-radius                : var(--fs-border-radius);
+
+  --fs-checkbox-transition                   : border .2s ease, background-color .2s ease, box-shadow .2s ease;
+
+  --fs-checkbox-before-top                   : 0.188rem; // 3px
+  --fs-checkbox-before-width                 : 0.375rem; // 6px
+  --fs-checkbox-before-height                : 0.625rem; // 10px
+  --fs-checkbox-before-border                : var(--fs-border-width-thick) solid var(--fs-color-neutral-0);
+  --fs-checkbox-before-border-radius         : var(--fs-border-width);
+  --fs-checkbox-before-transform             : rotate(45deg);
+
+  --fs-checkbox-bkg-color-hover              : var(--fs-color-primary-bkg-light);
+  --fs-checkbox-border-color-hover           : var(--fs-color-text);
+  --fs-checkbox-shadow-hover                 : 0 0 0 var(--fs-border-width) var(--fs-border-color-active);
+
+  // Checked
+  --fs-checkbox-bkg-color-checked            : var(--fs-color-primary-bkg);
+  --fs-checkbox-border-color-checked         : var(--fs-border-color-active);
+  --fs-checkbox-bkg-color-checked-hover      : var(--fs-color-primary-bkg-hover);
+
+  // Disabled
+  --fs-checkbox-bkg-color-disabled           : var(--fs-color-disabled-bkg);
+  --fs-checkbox-border-disabled              : var(--fs-border-width) solid var(--fs-border-color-disabled);
+  --fs-checkbox-border-color-disabled        : var(--fs-border-color-disabled);
+  --fs-checkbox-text-color-disabled          : var(--fs-color-disabled-text);
```

Comment:
The `disabled` should appear right after `--fs-checkbox-`

Reply:
thanks for the suggestions, all were applied [here](https://github.com/vtex-sites/gatsby.store/pull/61/commits/3cead8685b11fad9aeaeae149ce5e3d3d82d5a5d).